dts/bindings/rng/nordic,nrf54lcracen-ctrdrbg.yaml

```
@@ -0,0 +1,14 @@
+# Copyright (c) 2025, Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    Nordic nRF54L CRACEN CTR DRBG based (Random Number Generator)
```

```suggestion
    Nordic nRF54L CRACEN CTR_DRBG based (Random Number Generator)
```
CTR_DRBG is the correct naming convention according to NIST SP800-90A

---

changed (in all places where it was written with a " " instead of "_")